index.html

```diff
@@ -11,35 +11,35 @@
   </head>
 
   <body>
-    <header>
+    <header class="page__header">
       <h1>Frederike Bründermann</h1>
-      <nav>
-        <a href="#kontakt">KONTAKT</a>
+      <nav class="page__header-nav">
+        <a href="#footer__kontakt">KONTAKT</a>
         <a href="#bio">BIOGRAFIE</a>
        </nav>
     </header>
 
     <main>
-      <picture>
+      <picture class="portrait">
             <source media="(max-width: 599px)" src="images/FrederikeamSchreibtisch.JPG"/>
             <source media="(min-width 600px)" src="images/frederikemeer.JPG">
             <img src="images/frederikemeer.JPG" alt=""/>
       </picture>
      
-      <div>
-      <section id="interessen">
+      <div class="main__details">
+      <section class="details__interessen">
```

Comment:
I would swap the section and div's. Also, `details` is a new BEM block.
```suggestion
      <section class="details">
      <div class="details__interessen">
```